.github/workflows/terraform.yml

```diff
@@ -0,0 +1,38 @@
+name: Terraform
+
+on:
+  push:
+    branches:
```

Comment:
change this to use manual dispatch. We may only deploy it once but we'll see how things go. No doubt we'll be running it a few times on this PR to make sure things are working well.